zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/AbstractSchema.scala

```diff
@@ -199,26 +199,26 @@ abstract class Schema(typeName: String,
 
   def getTypeName: String = typeName
   def getTimeField: TimeField
```

Comment:
Since now the core difference between `Schema` and `LookupSchema` is the existence of `timeField`, then if it is false, we could know that this is a `LookupSchema`.